staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1613,7 +1613,7 @@ type VolumeProjection struct {
 }
 
 const (
-	ProjectedVolumeSourceDefaultMode int32 = 0644
+	ProjectedVolumeSourceDefaultMode int32 = 0444
```

Comment:
Is this needed? We should change this separately if not.

Reply:
no, i'll take it out.